code/modules/mob/living/simple_animal/hostile/megafauna/legion.dm

```diff
@@ -141,3 +141,80 @@ Difficulty: Medium
 	gpstag = "Echoing Signal"
 	desc = "The message repeats."
 	invisibility = 100
+
+/obj/structure/closet/crate/necropolis/legion
+	name = "Legion chest"
+	desc = "You feel the pressure of many thousand souls looking at you angrily."
+
+/obj/structure/closet/crate/necropolis/legion/New(add_loot = TRUE)
+	..()
+
+	if(!add_loot)
+		return
+
+	var/loot = rand(1, 26)
```

Comment:
Why is this rand 1, 26 when there is 27 items in this chest as options.

Reply:
Yeah that is a whoopsie I made. Fixed in bbab42d43a248a77477da047c878e483aa260114